cypress/integration/Pagination.spec.ts

```diff
@@ -376,6 +377,7 @@ describe('Pagination', () => {
         it('should go to the first page if a number below the range size is submitted', () => {
           cy.get('form')
             .find('input')
+            .type('{backspace}') // reset input value
```

Comment:
Because the initial value of the input is not model.state.currentPage, if you go into the input without backspacing or clearing the input, it adds a number to `1` so backspacing to clear it

Reply:
should we try clear instead?